lib/mayaUsd/ufe/EditAsMayaCommand.cpp

```diff
@@ -31,8 +33,6 @@ EditAsMayaUfeCommand::EditAsMayaUfeCommand(const Ufe::Path& path)
 {
 }
 
-EditAsMayaUfeCommand::~EditAsMayaUfeCommand() { }
```

Comment:
Empty override destructors are not needed. Also removed from .h. The macros test that the destructor is virtual.